src/lib/drupal/drupalClient.ts

```diff
@@ -62,10 +62,10 @@ export const drupalClient = new DrupalClient(baseUrl, {
   fetcher,
   useDefaultResourceTypeEntry: true,
   throwJsonApiErrors: false,
-  auth: {
-    clientId: process.env.DRUPAL_CLIENT_ID,
-    clientSecret: process.env.DRUPAL_CLIENT_SECRET,
-  },
+  // auth: {
+  //   clientId: process.env.DRUPAL_CLIENT_ID,
+  //   clientSecret: process.env.DRUPAL_CLIENT_SECRET,
+  // },
```

Comment:
noting we do need these values present for preview functionality to work. but maybe setting it here for everything is the wrong approach. instead we should probably include it during the specific calls for preview routes (eg lines 38-44 in `src/data/queries/newsStory.ts`)

Reply:
I commented this out just to get things to work. Forgot to remove it before pushing up.